content/_build/workers.html

```diff
@@ -0,0 +1,137 @@
+ 
+All binstar.org accounts allow you to attach one free worker. 
+If you need more workers you will need to [create a binstar
+organization](/#CreatingAnOrganization) and [Upgrade to a paid plan](/#PaidPlans).
+
+
+To follow along, you will need to 
+[install the build cli](/examples.html#InstallingTheBinstarCommand) 
+
+
+{%   call subsection('Create a build queue') %}
+
+The first thing you will need to do is create a build queue. 
+Build queues are used to as a container to direct builds you have submitted
+to the build workers you will create. Presently, when you submit a job the default build queue is
+named `binstar/public`.
+
+To create your queue run:
+{%   syntax bash %}
+binstar-build queue --create USERNAME/QUEUENAME
+{%   endsyntax %}
+
+You should replace `USERNAME` with your username and `QUEUENAME` with an alphanumeric name of your choice.
+
+By default a build worker will run builds submitted to your `binstar/public` you may change this one of two ways.
+
+ 1. Use the `--queue` option for the `binstar-build` [submit, save or trigger](cli.html#SubmittingBuilds) commands  
+    eg.
+    {% syntax bash %}
+    binstar-build submit ./my-build --queue USERNAME/QUEUENAME
+    {% endsyntax %}
+ 1. Set your default queue. 
+    This will affect all builds for your account. 
+    Navigate your browser to [binstar.org/settings/build-queue](https://binstar.org/settings/build-queue)
+    and select **Set as Default** 
+ 
+{%   endcall %}
+
+{%   call subsection('Launching a Build Worker') %}
+
+The basic build worker runs on your machine (linux, osx or windows) as the current user 
+(see [security considerations](#SecurityConsiderations)) 
+and accepts jobs from the build queue that you specify. 
+
+{% syntax bash %}
+binstar-build worker USERNAME/QUEUENAME
+{% endsyntax %}
+
+Thats it!  When you submit a job to your queue, your new build worker will pick it up and build it. 
+
+{%   endcall %}
+
+{%   call subsection('Share your build queue') %}
+
+Any build queues you create allow you to control who may submit jobs to it. 
+By default only the organization owners will be able to submit jobs.
+
+To share access to your queue:
+
+ 1. Navigate your browser to [binstar.org/settings/build-queue](https://binstar.org/settings/build-queue)
+ 1. Select the owner of the queue in the owner drop-down in the top right corner of the settings page.
+ 1. Click on the <i class="glyphicon glyphicon-pencil"></i> icon of the queue you want to share.
+ 1. Add the user by name or (individual accounts) or by group (organization accounts).
+
+
+{%   endcall %}
+
+
+
+{%   call subsection('Security Considerations') %}
+
+There are a few security considerations when launching a build worker.
+Remember the build worker runs user defined build scripts from the jobs that are submitted to it.
+
+You must:
+
+ 1. Consider who you are sharing your build queue with.
+ 1. Consider that `binstar-build worker` can run arbitrary scripts, make sure users can not accidentally 
```

Comment:
Can you clarify what you mean by "change the state of your build machine"? This doesn't automatically mean anything to me; I suspect other users might be similarly unaware.